gr-fft/python/fft/qa_fft.py

```diff
@@ -273,6 +273,20 @@ def test_reverse_window_shift(self):
         result_data = dst.data()
         self.assert_fft_ok2(expected_result, result_data)
 
+    def _set_length_and_test(self, fftlength: int, windowlength: int) -> bool:
+        win = [1.0] * windowlength
+        for shift in (True, False):
+            fft_block = fft.fft_vcc(fftlength, forward=True, window=[], shift=shift)
+            return fft_block.set_window(win)
+
+    def test_303_test_window_setter(self):
+        """
+        Test whether the window setter correctly returns False for lengths != FFT length.
+        """
+        for fftlength in [1, 2, 3] + [2**i for i in range(2, 10)] + list(primes):
+            self.assertTrue(self._set_length_and_test(fftlength, fftlength))
+            self.assertFalse(self._set_length_and_test(fftlength, fftlength+2))
```

Comment:
Not sure if will solve `./gr-fft/python/fft/qa_fft.py:288:76: E226 missing whitespace around arithmetic operator` but probability it will, according to [E226](https://www.flake8rules.com/rules/E226.html)
```suggestion
            self.assertFalse(self._set_length_and_test(fftlength, fftlength + 2  ))
```

Reply:
that will definitely cause another error due to superfluous whitespace *before* the `)`, @StudHamza  ;)